lib/concurrent/executor/ruby_thread_pool_executor.rb

```diff
@@ -296,8 +296,8 @@ def kill
 
       private
 
-      def create_worker(queue, pool, idletime)
-        Thread.new(queue, pool, idletime) do |queue, pool, idletime|
+      def create_worker(_queue, _pool, _idletime)
+        Thread.new(_queue, _pool, _idletime) do |queue, pool, idletime|
```

Comment:
Not sure what to do here, but the method signature is more important I think, so it should keep the names without `_`.